### ISSUE TITLE
JSON fields

### ISSUE THREAD

**Issue by dvv** [mabrek/epgsql#2]
Hi!

Wonder if there are any plans to make JSON fields (quite useful since 9.3) automatically encoded/decoded (jsx?).

TIA,
--Vladimir

**Comment by mabrek** [comment]
No. Database driver library shouldn't enforce any json decoder/encoder choice.

There might be possible to provide pluggable type encoder/decoders in the future, but not now. I'll leave this issue open to remind about it.

**Comment by dvv** [comment]
I expected that. Thanks.
